model/src/main/proto/topic.proto

```diff
@@ -239,6 +239,55 @@ message TopicProgress {
 
   // Map from story ID to StoryProgress.
   map<string, StoryProgress> story_progress = 2;
+
+  // Timestamp to record last time the exploration was played in ms.
+  int64 last_played_timestamp = 4;
+}
+
+// A structure corresponding to the Promoted Stories. This structure is set up
+// to properly account for recently_played stories, for recommended stories
+// and for coming soon topics.
```

Comment:
Please update documentation since the structure has changed.

Reply:
Updated design doc.